devel/phosg/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               cmake 1.1
+PortGroup               github 1.0
+
+github.setup            fuzziqersoftware phosg e1bd000
+revision                1
```

Comment:
I originally set a revision because I was only setting a commit hash to checkout. If I now set the version number as the commit date, should I hold off on setting a revision until it's needed?

Reply:
> If I now set the version number as the commit date, should I hold off on setting a revision until it's needed?

Setting the revision number explicitly as zero should be good.

Reply:
`revision 0` in both